src/main/java/flaxbeard/immersivepetroleum/api/crafting/LubricatedHandler.java

```diff
@@ -46,6 +46,8 @@
 		
 		void lubricate(World world, int ticks, E mbte);
 		
+		void lubricate(World world, int ticks, E mbte, AutoLubricatorTileEntity lubricator);
```

Comment:
I don't see why that would be needed and what for (it's always null in the current state anyway)

Reply:
I think i'd make more sense to just pass in the lubricant itself and not the whole lubricator.
ie `void lubricate(World world, int ticks, E mbte, FluidStack lubricant);`

Reply:
I figured that initially, but also figured if anyone else did anything they may want the te. But is definitely more efficient to pass the FluidStack, i can change that as well, it doesnt change anything for me.